[Chore] Update Readme (#53)
nimblehq/showcase-flutter-build-web-desktop: README.md

Reviewer: > Create these `.env` files in the root directory and add the required
  environment variables into them.

Shall we add an example of the required fields for `.env` file?

Author: What about something like this?

```env
REST_API_ENDPOINT=
NIMBLE_CX_USERNAME=
NIMBLE_CX_PASSWORD=
ACCESS_TOKEN=
```

Reviewer: @Wadeewee Please correct me if I am wrong, I have the following environment variables 👇 . Did you intentionally remove the rest? 🙏  The idea is for the developers to know which environment variables are needed later.  
```
CLIENT_ID=
CLIENT_SECRET=
REST_API_ENDPOINT=
USERNAME=
PASSWORD=
NIMBLE_CX_USERNAME=
NIMBLE_CX_PASSWORD=
ACCESS_TOKEN=
```

# Showcase flutter build web desktop

Building Web and Desktop (Windows) apps with Flutter:

- Welcome screen:

  <img width="1049" alt="Screen Shot 2566-07-27 at 09 39 20" src="https://github.com/nimblehq/showcase-flutter-build-web-desktop/assets/28002315/0900f56b-889f-4a0b-90f9-2044d377e819">

- Main screen:

>   <img width="1049" alt="Screen Shot 2566-07-27 at 11 12 50" src="https://github.com/nimblehq/showcase-flutter-build-web-desktop/assets/28002315/e76393f8-65f1-4d76-b5bb-7db46941fa3a">

Reviewer 2: Do we really need to set the width for the images here? Can't we leave them auto-scaled? 🤔

Author: Let me check @minhnimble 🏃‍♀️

>It provides additional information and troubleshooting steps specifically for resolving these issues
>encountered during local development.

### How to use the app:

1. Open the app and click on the `Sign in with CX` button.
2. On the main screen, select products to add to your cart.
3. On the right side of the main screen in the cart section:

>    - You can click on bin button to remove an item from the cart list.

Reviewer 2: Wording highlight improvement:
```suggestion
   - You can click the `Bin` button to remove an item from the cart list.
```

> 3. On the right side of the main screen in the cart section:
>    - You can click on bin button to remove an item from the cart list.

Reviewer 2: Can we shorten these sentences into 1 step like others? For example:
```suggestion
3. To remove a product from the cart list in the cart, select the `Bin` button on the right side of that product.
```
